### PR TITLE
Update widgets example

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

**Review by SaschaMann — APPROVED** [review]
(though I guess the file and header should be renamed to "widgets"?)

**Comment by ErikSchierboom** [comment]
I'll do that next week

**Comment by SaschaMann** [comment]
I was thinking we could have both examples to show slightly different uses

**Review by SleeplessByte — APPROVED** [review]

File: anatomy/tracks/widgets.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Widgets
+
+To allow rendering widgets for concepts and exercises on Exercism, a special type of Markdown link can be used. Instead of displaying a link, we'll render a user-contextual widget, that shows that concept or exercise with its icon, user-specific status (unlocked, completed, etc), and which provides a tooltip. 
+
+## Usage
+
+Widgets can be used wherever you can use Markdown, such as documentation, approaches and student/mentor discussions.
```
Comment (SaschaMann) [inline anatomy/tracks/widgets.md:7]:
```suggestion
Widgets can be used wherever you can use Markdown, such as documentation, approaches and student/mentor discussions.
The link reference, i.e. the contents of `()`, is ignored by the website.
You can point them at whatever you want for maintenance purposes.
```

Comment (ErikSchierboom) [reply anatomy/tracks/widgets.md:7]:
```suggestion
Widgets can be used wherever you can use Markdown, such as documentation, approaches and student/mentor discussions.

The link reference, i.e. the contents of `()`, is ignored by the website. You can point them at whatever you want for maintenance purposes.
```

Comment (ErikSchierboom) [reply anatomy/tracks/widgets.md:7]: Thanks for the suggestion!

**Comment by ErikSchierboom** [comment]
> I was thinking we could have both examples to show slightly different uses

Could you add a suggestion?

File: anatomy/tracks/widgets.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+The link reference, i.e. the contents of `()`, is ignored by the website. You can point them at whatever you want for maintenance purposes.
+
+### Concept widget
+
+Link format: `[concept:<track-slug>/<concept-slug>]()`
+
+Example:
```
Comment (SaschaMann) [inline anatomy/tracks/widgets.md]:
```suggestion
#### Examples
```

File: anatomy/tracks/widgets.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+```markdown
+There are three primary conditional statements that are used in Julia:
+
+- [concept:julia/if-statements]()
+- [concept:julia/ternary-operator]()
+- [concept:julia/short-circuiting]()
+```
```
Comment (SaschaMann) [inline anatomy/tracks/widgets.md:23]:
````suggestion
##### List of concept widgets

This list is in the middle of a file, embedded in prose with further text below it.

```markdown
There are three primary conditional statements that are used in Julia:

- [concept:julia/if-statements]()
- [concept:julia/ternary-operator]()
- [concept:julia/short-circuiting]()

[...]
```
````

File: anatomy/tracks/widgets.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
```
Comment (SaschaMann) [inline anatomy/tracks/widgets.md:24]:
````suggestion

##### Related concepts sections

This is a section with solely buttons at the end of a document.

```markdown
A ternary expression is an alternative to `if` statements.

## Related concepts

- [concept:ruby/if-statements]()
```
````

**Comment by ErikSchierboom** [comment]
I'll finish this up next week.

**Review by SaschaMann — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
I'll merge this just to prevent people from reading the old documentation. Adding screenshots and URLs is something I plan on doing later.

**Pull request merged**